src/app-init/router.js

```diff
@@ -18,6 +18,7 @@ export const ROUTE_WIDGET_EDIT = 'widgetEdit';
 export const ROUTE_FRAGMENT_ADD = 'fragmentAdd';
 export const ROUTE_FRAGMENT_EDIT = 'fragmentEdit';
 export const ROUTE_FRAGMENT_DETAIL = 'fragmentDetail';
+export const ROUTE_DATA_ADD = 'dataAdd';
```

Comment:
Please rename ROUTE_DATA_ADD to ROUTE_DATA_MODEL_ADD, as we'll also have ROUTE_DATA_TYPE_ADD.
Also please rename "dataAdd" to "dataModelAdd"